src/modules/dashboard/components/Members/MembersSection.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useCallback } from 'react';
+
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { ColonyRole } from '@colony/colony-js';
+import styles from './MembersSection.css';
+
+import MembersList from '~core/MembersList';
+import { AnyUser, Colony, useMembersSubscription } from '~data/index';
+
+import UserPermissions from '~dashboard/UserPermissions';
+import LoadMoreButton from '~core/LoadMoreButton';
+
+const displayName = 'dashboard.MembersSection';
+
+type Member = AnyUser & {
+  roles: ColonyRole[];
+  directRoles: ColonyRole[];
+  banned: boolean;
+};
+
+interface Props {
+  title: MessageDescriptor;
+  description?: MessageDescriptor;
+  sortingParams?: string[];
+  colony: Colony;
+  currentDomainId: number;
+  members: Member[];
+}
+
+const ITEMS_PER_SECTION = 10;
+const MembersSection = ({
+  title,
+  description,
+  colony: { colonyAddress },
+  colony,
+  currentDomainId,
+  members,
+}: // @NOTE Add another optional paramater called sortingOptions to handle sorting
+Props) => {
+  const [dataPage, setDataPage] = useState<number>(1);
+
+  /*
+   * @NOTE Poor man's pagination
+   *
+   * We're not really doing proper pagination like we do for say... `ColonyEvents`,
+   * because our metadata server doesn't support skip, first, more, next, etc...
+   * query filters
+   *
+   * A proper solution for this would be to teach the server to return just part
+   * of the results, with the next waiting in line for a callback, but the current
+   * timeframe doesn't allow for this, so I guess this is the "best" we can do for now
+   */
+
+  const paginatedMembers = members.slice(0, ITEMS_PER_SECTION * dataPage);
+  const handleDataPagination = useCallback(() => {
+    setDataPage(dataPage + 1);
+  }, [dataPage]);
+
+  const { loading: loadingAllMembers } = useMembersSubscription({
+    variables: {
+      colonyAddress,
+    },
+  });
+  return (
+    <>
+      <div className={styles.bar}>
+        <div className={styles.title}>
+          <FormattedMessage {...title} />
+        </div>
+        {description && (
+          <div className={styles.description}>
+            <FormattedMessage {...description} />
+          </div>
+        )}
+      </div>
+      <MembersList<Member>
+        colony={colony}
+        extraItemContent={({ roles, directRoles, banned }) => (
+          <UserPermissions
+            roles={roles}
+            directRoles={directRoles}
+            banned={banned}
+          />
+        )}
+        domainId={currentDomainId}
+        users={paginatedMembers}
+      />
```

Comment:
So the members list needs to be refactored to account for the new spec changes.

You cannot simply reuse it in this instance.